src/dso_api/middleware.py

```diff
@@ -24,7 +24,6 @@ def __call__(self, request: HttpRequest):
             # get_token_scopes is a data attribute, not a method.
             scopes = request.get_token_scopes or []
             request.user_scopes = UserScopes(request.GET, scopes, self._all_profiles)
-
```

Comment:
Unnecessary change, please revert to keep the patch smaller.

Reply:
done